src/test/java/uk/gov/hmcts/ccd/data/casedetails/search/SortOrderQueryBuilderTest.java

```diff
@@ -1,80 +0,0 @@
-package uk.gov.hmcts.ccd.data.casedetails.search;
-
```

Comment:
Looks good overall, but couldn't grasp the reason to remove this test class at first sight.